lib/views/variant-view.dart

```diff
@@ -76,84 +76,95 @@ class SortablePage extends StatefulWidget {
 }
 
 class _SortablePageState extends State<SortablePage> {
-  List<String> headerLabel = ['accession', 'geographical location', 'date collected', 'generated', 'pinned'];
+  List<String> headerLabel = ['accession', 'geographical location', 'date collected', 'generated', 'pinned', 'selected'];
```

Comment:
Also how hard would it be to move the selected box to the left side - feel like that's more intuitive

Reply:
Fixed

Reply:
I agree with Ben here, but this is probably a good point for future discussion